### PR TITLE
add metric tag name to lower-case test

### PR DESCRIPTION
**Pull request by KutuzovGregory**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/metric/MetricUpdateTest.java [hunk 1]
```diff
@@ -47,5 +51,48 @@ public void testUnknownMetric() throws Exception {
...
+    public void testMetricTagNameIsLowerCased() throws Exception {
+        final String FIRST_TAG_NAME = "SoMeTaG";
+        final String SECOND_TAG_NAME = "NeWtAg";
+        final String TAG_VALUE = "value";
+
+        Metric metric = new Metric("metric-with-tag");
+        Map<String, String> tags = new HashMap<>();
+        tags.put(FIRST_TAG_NAME, TAG_VALUE);
+        metric.setTags(tags);
+        Response response1 = createOrReplaceMetric(metric);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/metric/MetricUpdateTest.java]: we have `createOrReplaceMetricCheck` for this case

**Comment by Sild** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/metric/MetricUpdateTest.java [hunk 2]
```diff
@@ -47,5 +51,48 @@ public void testUnknownMetric() throws Exception {
...
+        assertEquals("Wrong metric tags count", updatedMetric.getTags().size(), 2);
+        assertTrue("Metric has not tag '"+FIRST_TAG_NAME.toLowerCase()+"'",
+                updatedMetric.getTags().containsKey(FIRST_TAG_NAME.toLowerCase()));
+        assertEquals("Wrong tag 'sometag' value",
+                updatedMetric.getTags().get(FIRST_TAG_NAME.toLowerCase()), TAG_VALUE);
+        assertTrue("Metric has not tag '" + SECOND_TAG_NAME.toLowerCase()+"'",
+                updatedMetric.getTags().containsKey(SECOND_TAG_NAME.toLowerCase()));
+        assertEquals("Wrong tag '"+SECOND_TAG_NAME.toLowerCase()+"' value",
+                updatedMetric.getTags().get(SECOND_TAG_NAME.toLowerCase()), TAG_VALUE);
+    }
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/metric/MetricUpdateTest.java]: please, split into 2 tests

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**